lab1/gen_ai_training/pom.xml

```diff
@@ -32,17 +46,20 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
spring-boot-starter-web to be replaced with:
```
 <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-webflux</artifactId>
 </dependency>
```
   in case we want in unblocking e2e: